utils/auth.js

```diff
@@ -0,0 +1,32 @@
+const jwt = require("jsonwebtoken")
+const secretKey = process.env.JWT_SECRET_KEY
+const userDao = require("../models/userDao")
+
+const loginRequired = async (req, res, next) => {
+    try {
+        const accessToken = req.headers.authorization
+
+        if ( !accessToken ) {
+            const error = new Error('NEED_ACCESS_TOKEN')
+		    error.statusCode = 401
+		
+		    return res.status(error.statusCode).json({message: error.message})
+        }
+
+        const decoded = await jwt.verify(accessToken, secretKey);
+
+        const userId = await userDao.getUserById(decoded.id)
```

Comment:
decoded라는 변수명 보다는 decoded해서 나온 결과물을 나타낼 수 있는 변수명을 사용하면 좋을 것 같습니다.
```suggestion
        const payLoad = await jwt.verify(accessToken, secretKey);

        const userId = await userDao.getUserById(payLoad.id)
```